api/controllers/WebActionsController.js

```diff
@@ -333,14 +340,12 @@ module.exports = {
                     platformId: resData.body.id,
                     referralCode: req.body.referredByCode,
                     referralCount: resData.body.referralCount,
+                    phone: referrerPhone,
```

Comment:
Is this getting added in just for the logging? Or is it also needed for what we're sending over to SNS?

If it's just for logging, I'd rather it get added into the log line alone as opposed to passing it on through the system.

Reply:
good call 👍🏻. i made the change